runelite-client/src/test/java/net/runelite/client/plugins/screenshot/ScreenshotPluginTest.java

```diff
@@ -36,8 +36,7 @@
 import net.runelite.api.events.GameTick;
 import net.runelite.api.events.WidgetLoaded;
 import net.runelite.api.widgets.Widget;
-import static net.runelite.api.widgets.WidgetID.DIALOG_SPRITE_GROUP_ID;
-import static net.runelite.api.widgets.WidgetID.LEVEL_UP_GROUP_ID;
+import static net.runelite.api.widgets.WidgetID.*;
```

Comment:
These changes still aren't reverted. Don't use star imports here. The full diff should show no differences in this test file.